src/utils/cmdline_text.ts

```diff
@@ -31,20 +31,6 @@ export function calculateInputAfterTextChange(oldText: string, newText: string):
     }
 }
 
-export function commandInputIsCompletable(command: string): boolean {
-    return (
-        command.charAt(0) !== "?" &&
-        command.charAt(0) !== "/" &&
-        !command.includes("s/") &&
-        !command.includes("su/") &&
```

Comment:
I was suspicious about this, it looked like a fragile hack. Good to eliminate it.

Reply:
+1, it's not great but it covers a lot of common cases